### ISSUE TITLE
Spurious warnings when running tox pylint over pylint on Mac

### ISSUE THREAD

**Issue by rogalski** [PyCQA/pylint#1505]
When `tox pylint` is run over Pylint itself on my Mac machine following errors occur:

```
************* Module pylint.checkers.format
W:1011, 0: Anomalous Unicode escape in byte string: '\u'. String constant might be missing an r or u prefix. (anomalous-unicode-escape-in-string)
W:1011, 0: Anomalous Unicode escape in byte string: '\u'. String constant might be missing an r or u prefix. (anomalous-unicode-escape-in-string)
************* Module pylint.checkers.spelling
C: 22, 4: Old-style class defined. (old-style-class)
W: 22, 4: Class has no __init__ method (no-init)
************* Module pylint.checkers.utils
W: 24, 4: Relative import 'backports.functools_lru_cache', should be 'pylint.checkers.backports.functools_lru_cache' (relative-import)
************* Module pylint.checkers.variables
W: 22, 4: Relative import 'backports.functools_lru_cache', should be 'pylint.checkers.backports.functools_lru_cache' (relative-import)
C: 29, 0: Imports from package pylint are not grouped (ungrouped-imports)
```

- [ ] investigate what triggers those errors and why they are no triggered in CI environment
- [ ] act accordingly, based on found root cause

### pylint --version output
master

**Comment by rogalski** [comment]
does not occur anymore
